mantidproject/mantid
# Fix intradocumentation links

**Description of work.**

This fixes broken links and some bad linking practices in the sphinx documentation. Rather than hard-linking to pages on the web or using relative paths, we should use reST labels and references and let sphinx take care of the linking.

**To test:**

Careful proofreading. Build the `docs-html` target and see if you can find more broken links, especially in sections considering fitting.

Fixes #23609.

*This does not require release notes* because **this does not introduce functional changes**.

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
